howto.md

```diff
@@ -1,14 +1,31 @@
 ---
-title: How to use them
+title: How to Run a Delta Chat Bot
 ---
 
-# How to use
+# How to Run a Delta Chat Bot
+
+To run a bot, you don't need to be a programmer. 
+You don't even need to own or rent a server.
+A bot runs on any computer.
+
+Any bot has a documentation about how to use it - you need to find this information.
```

Comment:
please don't do such incomplete sentences. I this form that sentence is really confusing. Especially the part 
> you need to find this information

sounds super weird.

Maybe something like this:
```suggestion
All of our example bots feature a documentation on how to use them and most of our bots have comments in the source code to document how they work.
```

Reply:
I disagree, it was not an incomplete sentence. Both sentences have a subject, predicate, and object, and are separated by a hyphen, which is a very normal sentence structure^^ 

But I changed it nonetheless, and also added that you can easily find it in the README.md.